public/services/CorrelationService.ts

```diff
@@ -54,6 +59,9 @@ export default class CorrelationService implements ICorrelationsService {
 
     return (await this.httpClient.post(url, {
       body: JSON.stringify(body),
+      query: {
```

Comment:
NIT: we can make a function to create a query object, since we are using this everywhere

Reply:
Let me look at this in a follow up PR since it doesn't affect the logic